### PR TITLE
docs: adds example app docs

### PR DESCRIPTION
**Pull request by riyalohia**
### What does this implement/fix? Explain your changes.
Adds covid app tutorial

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Review by satyamyadav — COMMENTED** [review]
The pull request type should be `docs`.

File: docs/AppTutorial.md [hunk 1]
```diff
@@ -0,0 +1,461 @@
+# COVID-19 Tracker App Tutorial
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: Heading should be `Tutorial` only, and `Covid-19 Tracker App` as a subheading.

File: docs/AppTutorial.md [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+
+Welcome! This tutorial will guide you in creating an awesome app with Innovaccer Design System. Here is a [preview](https://covid19-tracker.info/) of the app that we are going to build. This tutorial also has an accompanying [github repository](https://github.com/sandeshchoudhary/covid).
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: Please add linebreak after `Welcome!`.

File: docs/AppTutorial.md [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+
+## Audience
+This tutorial is intended for people with all amounts of web development experience. If you want to jump straight to code, you may want to skip this tutorial and go to the [developer guidelines](https://github.com/aregee/design-system/blob/master/README.md) page.
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: The link should be https://github.com/aregee/design-system/blob/master/CONTRIBUTING.md or https://github.com/aregee/design-system/#-how-to-contribute

File: docs/AppTutorial.md [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+
+## Prerequisites
+* Node JS
+* NVM
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: Mentioning NVM is not needed.

File: docs/AppTutorial.md [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+
+## Getting Started
+Each step in this guideline illustrates a different aspect of developing web applications with Innovaccer design system. Refer the following sections below to get familiar with this project:
+* [Installation](#Installation)
+* [Initialising app](#APP_INIT)
+* [Building Landing Page](#BUILDING_LANDING_PAGE)
+* [Building Drilled Page](#BUILDING_DRILLED_PAGE)
+
+
+## <a name="Installation"></a>Installation
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: This title should be `Initializing App`.

File: docs/AppTutorial.md [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+We will start with `Create React App`.
+```jsx
+npx create-react-app covid
+cd covid
+npm start
+```
+Next, you need to install Innovaccer design system in your project. You need to follow the [developer guidelines](https://github.com/aregee/design-system/blob/master/README.md) steps to install the design system.
+
+## <a name="APP_INIT"></a>Initialising app
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: The title should be `Installing component Library`.

File: docs/AppTutorial.md [hunk 7]
```diff
@@ -0,0 +1,461 @@
...
+
+
+## <a name="Installation"></a>Installation
+We will start with `Create React App`.
+```jsx
+npx create-react-app covid
+cd covid
+npm start
+```
+Next, you need to install Innovaccer design system in your project. You need to follow the [developer guidelines](https://github.com/aregee/design-system/blob/master/README.md) steps to install the design system.
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: Add npm install command for library and then provide the link as to refer for more details.

File: docs/AppTutorial.md [hunk 8]
```diff
@@ -0,0 +1,461 @@
...
+
+## <a name="APP_INIT"></a>Initialising app
+A preview of what you will build:
+
+![alt tag](./images/Covid-App.png)
```
Comment (satyamyadav) [inline docs/AppTutorial.md:42]: Move preview under first title (introductory part).

File: docs/AppTutorial.md [hunk 9]
```diff
@@ -0,0 +1,461 @@
...
+Then, update the render() function to include the Router.
+```jsx
+<div>
+    <Header />
+    <Router basename={process.env.PUBLIC_URL}>
+        <Footer />
+    </Router>
+</div>
+```
+## <a name="BUILDING_LANDING_PAGE"></a>Building Landing Page
```
Comment (satyamyadav) [inline docs/AppTutorial.md]: Heading should be something like `Creating page grid` or `Adding grid to page` and then in description highlisht that we are building first page of app.

File: docs/AppTutorial.md [hunk 10]
```diff
@@ -0,0 +1,461 @@
...
+    )
+    ```
+* ###### Fetch data
+    Next we’ll assemble our GraphQL using the gql helper. The gql helper lets you write GraphQL queries using interpolated strings (backticks) in JavaScript. In addition, we’ll be using the Query component from react-apollo which gives us some great information about our query’s loading state in addition to the data. [Click here](https://github.com/sandeshchoudhary/covid/blob/master/src/query.js) for more details. 
+
+Finally, you can get the data, loading and error states in your component using `useQuery` from `@apollo/react-hooks`. For example: 
+```jsx
+const { loading, error, data } = useQuery(query.data);
+```
+Now that we have fetched the required data, we can easily render them in our component. Now its time to build `Map component` using the fetched API data. Here is the preview of what you will build:
```
Comment (satyamyadav) [inline docs/AppTutorial.md:236]: Highlight or add a heading to inform we are going to use visualization components.

**Review by satyamyadav — CHANGES_REQUESTED** [review]

File: docs/AppTutorial.md [hunk 11]
```diff
@@ -0,0 +1,472 @@
...
+```jsx
+<header>
+    <Heading size="l" appearance="white">
+        Covid-19
+    </Heading>
+</header>
+```
+Now we will create a Footer for our app. Here is a preview of Footer component:
+
+![alt tag](./images/Footer.png)
```
Comment (aditya-kumawat) [inline docs/AppTutorial.md:67]: Either update the image or code below it

File: docs/AppTutorial.md [hunk 12]
```diff
@@ -0,0 +1,472 @@
...
+```
+Now, we need to wrap our app in the Router component. In the root *index.js*, add the import:
+```jsx
+import { HashRouter as Router, Switch, Route } from 'react-router-dom';
+```
+Then, update the render() function to include the Router.
+```jsx
+<div>
+    <Header />
+    <Router basename={process.env.PUBLIC_URL}>
```
Comment (aditya-kumawat) [inline docs/AppTutorial.md]: I guess process.env.PUBLIC_URL is for github pages. Replacing it with '/' would be better.

File: docs/AppTutorial.md [hunk 13]
```diff
@@ -0,0 +1,472 @@
...
+```
+Note that we are fetching *currentMap*, *currentMapData* etc. from API. Now moving to our second column, we can add map info cards in the following way:
+```jsx
+ <div className="Map-info-item">
+    <div>
+        <div className="InfoLegend"></div>
+        <Text>Confirmed</Text>
+    </div>
+    <div className="pt-3 pl-5">
+    <Text style={{ fontSize:'28px'}}>
```
Comment (aditya-kumawat) [inline docs/AppTutorial.md]: Remove style
Not recommended.

File: docs/AppTutorial.md [hunk 14]
```diff
@@ -0,0 +1,472 @@
...
+
+Lets add a basic grid.
+```jsx
+import {Row, Column} from 'design-system';
+```
+```jsx
+<Row>
+    <Column>...</Column>
+    <Column>...</Column>
+</Row>
```
Comment (aditya-kumawat) [inline docs/AppTutorial.md]: Update with correct code.

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**